### ISSUE TITLE
Missing labels / accessible names for input and select elements

### ISSUE THREAD

**Issue by aviannamiller** [omeka/plugin-ItemRelations#21]
A few missing labels / names on item edit:

- Text `input` `item_relations_item_relation_object_item_id` is missing a label.
- `select` element `item_relations_property_id` is missing an accessible name.

**Comment by aviannamiller** [comment]
On Map -> Browse:
- `item_relations_property_id` select is missing an accessible name.
- `item_relations_clause_part` radio inputs are missing labels and a grouping treatment.

**Comment by aviannamiller** [comment]
Edit Custom Vocabulary:
- All textarea and checkbox inputs are missing labels.

**Comment by aviannamiller** [comment]
On Items -> Search:

- `object` and `subject` radio inputs are missing labels.
- `item_relations_property_id` select is missing a name.
